src/components/profile/ProfileTabs.vue

```diff
@@ -0,0 +1,99 @@
+<script lang="ts">
+/**
+ * ProfileTabs Component
+ *
+ * @description * Use this component to render tabs on the Profile page.
+ * Note: Used on `ProfilePage`.
+ *
+ * @components
+ *
+ * @example
+ * <profile-tabs />
+ *
+ * @see [Figma Design](https://www.figma.com/design/L8dVREySVXxh3X12TcFDdR/Do-pr%C3%A1ce-na-kole?node-id=4858-104393&t=X2rh1VOnQ1CkvTqG-1)
+ */
+
+// libraries
+import { defineComponent, ref } from 'vue';
+
+// routes
+import { routesConf } from 'src/router/routes_conf';
+
+type ProfileTabsOption = 'details' | 'forms' | 'newsletter' | 'notifications';
+
+export default defineComponent({
+  name: 'ProfileTabs',
+  setup() {
+    const activeTab = ref('' as ProfileTabsOption);
+
+    return {
+      activeTab,
+      routesConf,
+    };
+  },
+});
+</script>
+
+<template>
+  <div>
+    <!-- Tab buttons -->
+    <q-tabs
+      inline-label
+      v-model="activeTab"
+      class="text-grey"
+      active-color="primary"
+      indicator-color="primary"
+      align="center"
+      data-cy="profile-tabs"
+    >
+      <q-route-tab
+        :to="routesConf['profile_details'].path"
+        name="details"
```

Comment:
`name` prop value should be TS Enum.

Reply:
Used TS Enum.

Reply:
You still use string literal `name="details"`.

Reply:
Sorry, misread the correct point in the code. Fixed now.